src/tables/cmap/format0.rs

```diff
@@ -15,3 +15,46 @@ pub fn parse(data: &[u8], code_point: u32) -> Option<u16> {
         None
     }
 }
+
+pub fn codepoints(data: &[u8], mut f: impl FnMut(u32)) -> Option<()> {
+    let mut s = Stream::new(data);
+    s.skip::<u16>(); // format
+    s.skip::<u16>(); // length
+    s.skip::<u16>(); // language
+
+    for code_point in 0..256 {
+        // In contrast to every other format, here we take a look at the glyph
+        // id and check whether it is zero because otherwise this method would
+        // always simply call `f` for `0..256` which would be kind of pointless
+        // (this array always has length 256 even when the face has only fewer
+        // glyphs).
+        let glyph_id: u8 = s.read()?;
+        if glyph_id != 0 {
+            f(code_point);
+        }
+    }
+
+    Some(())
+}
+
+#[cfg(test)]
+mod format0_tests {
```

Comment:
`tests` are enough. No need to use the `formatN` prefix.

Reply:
Okay, I did that because it was like that for format 4 but I can change it.